build/build/src/enso.rs

```diff
@@ -145,7 +145,12 @@ impl BuiltEnso {
         let may_need_postgres = match &test_selection {
             StandardLibraryTestsSelection::All => true,
             StandardLibraryTestsSelection::Selected(only) =>
-                only.iter().any(|test| test.contains("Postgres_Tests")),
+                only.iter().any(|test| test.contains("Table_Tests")),
+        };
+        let may_need_sqlserver = match &test_selection {
+            StandardLibraryTestsSelection::All => true,
+            StandardLibraryTestsSelection::Selected(only) =>
+                only.iter().any(|test| test.contains("SQLServer")),
```

Comment:
I think this is not right, the project is called `Microsoft_Tests`, right?

```suggestion
                only.iter().any(|test| test.contains("Microsoft_Tests")),
```